python/tvm/relay/frontend/tensorflow.py

```diff
@@ -1171,6 +1172,100 @@ def _get_abs_layer_name(node):
                                                      params, num_layers)
         return sym
 
+_control_flow_nodes = ['Merge', 'Switch', 'NextIteration', 'Exit', 'Enter', 'LoopCond']
+
+class Branch:
```

Comment:
How about moving these ```Branch``` and ```Loop```  definitions to cpp ?
It helps the CPP frontend initiative in future.

If not now we should consider it later !

Reply:
yeah, we probably don't want to have that for now because there is no c++ parser yet. We can add it later.